components/clp-mcp-server/clp_mcp_server/validator/__init__.py

```
@@ -0,0 +1,3 @@
+from .validate import validate_query
```

Ruff (0.14.3)

Missing docstring in public package

(D104)